examples/output/Nearshore collection/bathymetry.json

```diff
@@ -1,6 +1,6 @@
 {
   "min": -1729,
   "max": 0,
-  "mean": -874.834883627011,
+  "mean": -874.7040171931454,
```

Comment:
These changes could be system specific, or may be due to an upgrade of geoblaze.